Cargo.toml

```diff
@@ -4,6 +4,9 @@ edition = "2021"
 repository = "https://github.com/paritytech/polkadot-sdk.git"
 license = "GPL-3.0-only"
 
+[workspace.dependencies]
+log = { version = "^0.4.20", default-features = false }
```

Comment:
I saw that if the default-features are not disabled in the workspace, then they cannot be disabled lateron in the crates, since features are additive.  
So AFAIK it is best practice to disable them at the workspace and then re-enable at the crate level - if desired. It is explained here https://github.com/rust-lang/cargo/issues/11329